WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -201,6 +198,36 @@ extension ProductInventorySettingsViewController {
 private extension ProductInventorySettingsViewController {
     func handleSKUChange(_ sku: String?) {
         self.sku = sku
+
+        // If the sku is identical to the old one, is always valid.
+        guard sku != product.sku else {
+            skuIsValid = true
+            hideError()
+            self.getSkuCell()?.textFieldBecomeFirstResponder()
```

Comment:
nit: `self` might not be needed?

Reply:
also, I think it might not be necessary to prompt the SKU field if it's valid?

Reply:
It's needed because if you pass from a not valid state to a valid state, you will see the keyboard closed, but maybe you haven't finished writing